src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -183,29 +188,31 @@ def _watch(self, res_ver):
 
 
 class RookCluster(object):
-    def __init__(self, coreV1_api, batchV1_api, rook_env):
+    # import of client.CoreV1Api must be optional at import time.
+    # Instead allow mgr/rook to be imported anyway.
+    def __init__(self, coreV1_api: 'client.CoreV1Api', batchV1_api: 'client.BatchV1Api', rook_env: RookEnv):
```

Comment:
```suggestion
    def __init__(self, coreV1_api: 'client.CoreV1Api', batchV1_api: 'client.BatchV1Api', rook_env: 'RookEnv'):
```

Reply:
or just define dummy classes for these types if `.rook_client._helper` and `rook.module` are not available?